apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -22,15 +26,43 @@ h2 + .c-card-wrapper {
     padding: 0;
 }
 
+.c-card-list--fillContainer {
+    grid-template-columns: 1fr;
+
+
+    @include f.media('>=mid') {
+        grid-template-columns: auto auto auto;
+
+        @container (#{$card-container-max-width}) {
+            li:last-child {
+                grid-column: span 2;
+            }
+        }
+
+    }
+}
 
 .c-card {
     position: relative;
     border: 1px solid f.$color-border-default;
     border-radius: f.$radius-rounded-c;
     background-color: f.$color-container-default;
+    padding: f.$spacing-e;
+    height: 100%;
```

Comment:
I'm certain this is something you specify in row spans. Is height 100% necessary? happy for it to stay in, just curious

Reply:
The grid is made up of `<li>` so this div needs to be 100% to fill that 🙂